### ISSUE TITLE
The output is not displayed in tabular form by using %%prun

### ISSUE THREAD

**Issue by ydmykr** [microsoft/vscode-jupyter#11443]
```
%%prun
print("hello")
```

![image](https://user-images.githubusercontent.com/100815370/192178724-b3a5e94a-1ba5-40f0-a70e-6464901584e8.png)

The output is very messy, not formatted or presented in tabular form.

**Comment by amunger** [comment]
thanks for the issue. Interesting to note that jupyter notebooks puts the output in a unique location:

<img width="866" alt="image" src="https://user-images.githubusercontent.com/2019016/192341684-3782cfae-e41b-41ca-a0f1-20e0177b57c4.png">

**Comment by DonJayamanne** [comment]
This is similar to the issue with HTML outputs not being aligned properly and not using monospaced font.
I found that using `display: block; white-space:pre-wrap;font-family: <some mono space like terminal font>` fixes this.

TLDR: I think we should re-visit the fonts used in the outputs, and must use a mono spaced font at least in stdout/stderr outputs, after all its output coming from a terminal.
@rebornix /cc

**Comment by ydmykr** [comment]
`display: block; white-space:pre-wrap;font-family: <some mono space like terminal font>`

I‘m so sorry that I don't quite understand this code, it looks like something used in css. How can I place them in jupyter?

**Comment by DonJayamanne** [comment]
@ydmykr please ignore that comment, it was meant for us (internal notes for the extension maintainers).

**Comment by robna** [comment]
maybe it is obvious to you but just in case:
the same occurs for the line magic `%prun`

@DonJayamanne do you have a workaround that could be applied from the user side, while it is not correctly formatted automatically? Would really appreciate!

**Comment by RomanCast** [comment]
The workaround I'm using at the moment is saving the `%prun` output to a separate file and inspecting that file.

`%prun -T tmp.txt my_fn()`

Thanks for looking into this !

**Comment by DonJayamanne** [comment]
@rebornix your new changes seems to have fixed this issue, 👍🏾 
I'm closing this as fixed in the latest vscode insiders version.
![Screenshot 2022-12-30 at 14 40 34](https://user-images.githubusercontent.com/1948812/210032199-a6322a08-1f6e-4549-9f3a-df739ee88ab2.png)

**Comment by starball5** [comment]
Related on Stack Overflow: [Jupyter %%prun magic output not displayed properly in VS Code](https://stackoverflow.com/q/73833016/11107541)
